src/main/java/com/revellr/core/partyservice/Models/College.java

```diff
@@ -0,0 +1,23 @@
+package com.revellr.core.partyservice.Models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+
+@Entity
+public class College{
```

Comment:
- add table annotation for clarity with table name
- add lombok `Data` annotation for Getter, Setter, ToString, EqualsAndHashCode, NoArgsConstructor, AllArgsConstructor, RequiredArgsConstructor.